### PR TITLE
(#17) added pushgateway sakuli configuration doc

### PR DESCRIPTION
**Pull request by cliffle**

### CODE REVIEW

File: docs/pushgateway-sakuli-configuration.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Sakuli Properties for prometheus pushgateway
+
+| Property name                         | Property description       |
+|---------------------------------------|----------------------------|
+| sakuli.forwarder.prometheus.api.host  | address of prometheus host |
+| sakuli.forwarder.prometheus.api.port  | port of prometheus host    |
+| sakuli.forwarder.prometheus.api.job   | value of the job label     |
```
Comment (tnobody) [inline docs/pushgateway-sakuli-configuration.md]: Looks fine 👍 
One question: is there any explanation on how the `job` label relates to the labels described in https://github.com/sakuli/sakuli-enterprise-forwarder/blob/develop/docs/prometheus-mapping.md or what it does and means at all?

Comment (cliffle) [reply docs/pushgateway-sakuli-configuration.md]: There is no correlation between the `job` label and the labels described in the linked document. The `job` label is defined by the prometheus configuration. It defines when and where prometheus needs to scrape the data. You then can access it via the `job` label with `/metrics/job/<jobName>`. An example for a prometheus configuration is [here in line 31](https://github.com/stefanprodan/dockprom/blob/master/prometheus/prometheus.yml#L31).

Comment (tnobody) [inline docs/pushgateway-sakuli-configuration.md]:
```suggestion
| sakuli.forwarder.prometheus.api.job   | value of the job label, configured in the prometheus scrape configuration   |
```

What do you think to point that out a bit more?

Comment (cliffle) [reply docs/pushgateway-sakuli-configuration.md]: sounds good :+1:

**Review by tnobody — APPROVED** [review]

**Pull request merged**